python/tvm/auto_scheduler/relay_integration.py

```diff
@@ -34,18 +34,26 @@
 
 
 def call_all_topi_funcs(mod, params, target):
-    """Call all TOPI compute + schedule to extract tasks in a relay program"""
+    """Call all TOPI compute to extract auto_scheduler tasks in a Relay program"""
     # pylint: disable=import-outside-toplevel
     from tvm import relay
     from tvm.relay.backend import graph_runtime_codegen
 
-    with transform.PassContext(opt_level=3):
+    # Turn off AutoTVM config not found warnings
+    old_autotvm_silent = autotvm.GLOBAL_SCOPE.silent
+    autotvm.GLOBAL_SCOPE.silent = True
+
+    with transform.PassContext(opt_level=3, config={"relay.backend.use_auto_scheduler": True}):
```

Comment:
Given that there might be multiple backend arguments in the future, consider change to
```
config= {
   "relay.backend": {"use_auto_scheduler": True},
}
```

Reply:
Considering auto_scheduler would be a widely used feature, would this config be too long for most users to remember? Should we provide syntactic sugar, or maybe we can help users set this config in the `ApplyHistoryBest` like the following, although the order of setting build environments has to be fixed in this case.

```python
with transform.PassContext(opt_level3):
    with auto_scheduler.ApplyHistoryBest(log_file): # Set "relay.backend": {"use_auto_scheduler": True} to the current pass context?
        # ...
```

cc @merrymercy

Reply:
I see, I think we can go with the current proposal if that is the concern